File: assignment 10 - C# practice problems/20.cs
//Write a C# function to compare dates

using System;
namespace MyNameSpace
{
    class Program20
    {
        public static void CompareTwoDatesDriverFunction()
        {


            Console.WriteLine("Enter 2 dates in dd/mm/yyyy format");
            var dateInput1 = Console.ReadLine();
            var dateInput2 = Console.ReadLine();
            var split1 = dateInput1.Split('/');
            var split2 = dateInput2.Split('/');

            var date1Obj1 = new DateTime(int.Parse(split1[2]), int.Parse(split1[1]), int.Parse(split1[0]));
            var date1Obj2 = new DateTime(int.Parse(split2[2]), int.Parse(split2[1]), int.Parse(split2[0]));
            var result = DateTime.Compare(date1Obj1, date1Obj2);

            if (result == 0)
            {
                Console.WriteLine("Both dates are same");
            }
            else if (result > 0)
            {
                Console.WriteLine(dateInput2 + " is greater");
            }
            else
            {
                Console.WriteLine(dateInput1 + " is greater");
            }


        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/20.js
//Write a JavaScript function to compare dates
function compareTwoDates(dateParam1, dateParam2) {

    let date1 = new Date(dateParam1[2], dateParam1[1], dateParam1[0]);
    let date2 = new Date(dateParam2[2], dateParam2[1], dateParam2[0]);

    let date1Time = date1.getTime();
    let date2Time = date2.getTime();

    if (date1Time === date2Time) {
        alert("Both dates are same");
    } else if (date1Time > date2Time) {
        alert(dateParam1.join("/") + " is greater");
    } else {
        alert(dateParam2.join("/") + " is greater");
    }
}

function compareTwoDatesDriverFunction() {
    let date1 = prompt("Enter date1 in dd/mm/yyyy");
    let date2 = prompt("Enter date2 in dd/mm/yyyy");
    compareTwoDates(date1.split("/"), date2.split("/"));
}

compareTwoDatesDriverFunction();